File: src/main/java/com/iflat/ss/bean/FsCode.java
package com.iflat.ss.bean;

/**
 * Created by tyriv on 2016/8/5.
 */
public class FsCode {
    private String id;
    private String type;
    private String code;
    private String description;
    private double amount;
    private double score;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public double getScore() {
        return score;
    }

    public void setScore(double score) {
        this.score = score;
    }
}

File: web/app/model/ss/FsCode.js
Ext.define('iFlat.model.ss.FsCode', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'fsCode.id', mapping: 'id', type: 'string'},
        {name: 'fsCode.type', mapping: 'type', type: 'string'},
        {name: 'fsCode.code', mapping: 'code', type: 'string'},
        {name: 'fsCode.description', mapping: 'description', type: 'string'},
        {name: 'fsCode.amount', mapping: 'amount', type: 'number'},
        {name: 'fsCode.score', mapping: 'score', type: 'number'},
    ]
});